src/components/Button/OvalButton.js

```diff
@@ -0,0 +1,28 @@
+import * as React from "react";
+import { styled } from "@mui/material/styles";
+import { Button } from "@mui/material";
+import { DEFAULT_BUTTON_HEIGHT } from "../../contants/sizes";
+
+const OvalButtonStyled = styled(Button)(({ theme, color }) => {
+  return `
+  background: linear-gradient(45deg, ${theme.palette[color].alternative}, ${theme.palette[color].main});
```

Comment:
That's where I mentioned it can break the app
May you can something like:
```ts
${theme.palette[color].alternative || DEFAULT_ALTERNATIVE_COLOR}
```

And the same for the main color

Reply:
moved the background customization to the theme creation file. check the next pull request